base/src/numbers.rs

```diff
@@ -25,6 +25,12 @@ use wire_type::WireType;
 #[wire_type(recurse = 2)]
 pub struct Length(pub u32);
 
+impl From<u32> for Length {
```

Comment:
we can use [derive_more](https://github.com/JelteF/derive_more) if we want

Reply:
Good suggestion, fixed!